news/2021-06-14-mca-ayim-2019.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: Mapper's Choice Awards and A Year In Mapping 2019
+date: 2021-06-14 00:00:00 +0000
+---
+
+We're going to have a back to back experience for celebrating the 2019 and 2020 years of mapping!
+
+This June and July will be Mappers' Choice Awards _(MCA)_ and A Year in Mapping _(AYIM)_ for the 2019 season, and soon after that we will be hosting the same events but for 2020!
+
+It's been a while, but we're back with an epic pair of websites to allow for a better experience than ever.
+
+For now, we're all about celebrating 2019!
+
+## Schedule for MCA and AYIM 2019
+
+| Event | Timestamp (UTC) |
+| --: | :-- |
+| Nomination stage | 2021-06-14 00:00 to 2021-06-21 00:00 |
+| Voting stage | 2021-06-23 00:00 to 2021-07-07 00:00 |
+| Results Stream | 2021-07-10 / 2021-07-11 |
+| Results posted to Website | 2021-07-12 00:00 |
+
+## A Year In Mapping 2019
+
+AYIM is back at long last for another round of highlighting the achievements and statistics of the mapping community in 2019. As in previous editions, each mode will have a section of statistics such as who ranked the most maps, how many difficulties were ranked overall, the extremes in difficulty levels we saw ranked, and a whole lot more records than you can think of.
+
+**Here's what's new:** In previous years, organizers were responsible for writing comments on each mapper ranked that year. This time, we're opening it up to the whole community. Know some banger maps from 2019 or a mapper left a huge impression on you at that time? Let them know your thoughts by posting your comments to the website. This is open to _everyone_ to participate in, so let your positivty be heard!
```

Comment:
if possible maybe have some for of emphasis here like bolding and caps Lol or # heading, not sure whats the most appropriate

Reply:
Maybe also emphasize that anyone who has an osu! account can comment (this includes players and all)

Reply:
```suggestion
**Here's what's new:** In previous years, organizers were responsible for writing comments on each mapper ranked that year. This time, we're opening it up to the whole community. Know some banger maps from 2019 or a mapper left a huge impression on you at that time? Let them know your thoughts by posting your comments to the website. This is open to _everyone_ to participate in, so let your positivity be heard!
```